OVERVIEW.md

```diff
@@ -0,0 +1,124 @@
+
+### E6. Graph viewing ratios
+
+Our x-axis value is hard-coded. That means that if benefits taper off half-way through, half of the graph isn't useful. That also means the person looking at the graphs is always looking at the same size of graph. They can easily compare one set of life circumstances to another set. One feature we're working on is the ability to zoom in on the graph to see a section of particular interest more clearly.
+
+### E7. Usability
+
+The type of interaction our partners were looking for was a user-friendly one. Something less like Free Fillable Forms. We'd still like to give the user more guidance and information, but we currently don't have access to that information ourselves.
+
+
+
+## F. Next Steps
+
+- Develop a coding style for the calculation logic that mimics the structure of the regulations it represents, easing communication between subject matter experts and coders.
+- Develop a all-of/any-of/none-of structure for form questions to manage which questions are show to the user.
+- Accessibility, including:
+  - Useful tab order.
+  - Better contrast for UI responsiveness.
+  - aria-* attributes
+- Mobile-friendly styles.
+- More plain-language versions of explanations.
+- Implement zooming on the graphs of the results.
+- Better predictions - don't wait for the user to guess a future income amount. Instead, go through all future values and look for all possible cliffs.
+- Convert to a system where the app takes information about each individual household member, not just the household as one unit.
+- Allow people with permission to mark a benefit as out of date. Warn the user about that benefit's calculations.
+- Automated expiration dates for benefit data and logic that are used to let app users know when something may be less accurate.
+- Set up data tables that non-coders can edit to change how the benefits are calculated.
+- With the help of a subject matter expert, create logic that notes what client change has caused the main change in benefit amount, and what benefit rules caused the main change in the benefit amount.
+- Translations for non-English readers.
+- A pop-out 'calculator' that will show weekly, monthly, and yearly values for whichever you put in - basically the same as any money input row.
+- A pop-out 'calendar' for putting in the varying amounts of work for the year that will then average out the amounts for the year.
+- User-friendly icons and colors to more clearly visually represent what's going on.
+
+
+
+
+## G. Code and Code Cautions
+
+### G1. WARNING: Values the user shouldn't change
+
+The options for what circumstances the user can try to predict can be expanded. That said, never allow users to change certain kinds of future values. For example, a value like `future.rentShare` - they're not in charge of their subsidy. They can change `future.contractRent`, which is out of their control, but still something that the calculations aren't changing themselves.
```

Comment:
This is a little confusing - I think I know what you mean, but you could elaborate on why some things shouldn't be inputs. Inputting `rentShare` wouldn't make sense because that's what the app is trying to calculate, right? So a basic rule of thumb would be that a value can't be both an input and an output.

On the other hand, what if the client knew for sure what their future rent share would be, but wasn't sure about some of their other benefits? Could we have a way to override what the tool would normally output for a benefit, to affect the calculation of other benefits? Maybe that's too hypothetical a situation though.

Reply:
People do put in their current rent share - it's the only way we can have a chance at calculating their future rent share.

I do think that it should be possible for a client to put in exceptions and such because those do happen, but I do agree that that's a hypothetical right now.

Reply:
Changed the text a bit. Let me know what you think.

Reply:
^ right sorry, I meant future rent share.

Reply:
So the reason I put this section in is because it the rent subsidy is sort of an input and an output. It's an input for current values, but can't be an input for future values. Did that come through in the new text?

Reply:
Hmm, actually maybe I should put it more like that in the text of the overview...

Reply:
Right, I agree it's confusing and should be explained. Didn't mean to imply that it's obvious. I was just saying we should explain here that you can't put something as an input if it's already a calculated result from the app.

Reply:
I'll play around with the wording a bit more.